templates/prescription/list.twig

```diff
@@ -0,0 +1,38 @@
+{% extends 'layout.html.twig' %}
+{% block title %}{{ tabtitle }}{% endblock %}
+{% block body %}
+
+    <div class="container">
+    <h3>{{ pagetitle }}</h3>
+        <table width="100%">
+            <tr>
+                <td><a class="btn btn-primary" href="#">Print</a></td>
+                <td align="right">
+                    <a href="#" class="small" onClick="Check(document.presc.check_list);"><span> {{"Check All"|xlt}}</span></a> /
+                    <a href="#" class="small" onClick="Uncheck(document.presc.check_list);"><span>{{ "Clear All"|xlt }}</span></a>
+                </td>
+            </tr>
+        </table>
+        <table class="table" id="prescriptions">
+           <tr>
+               <th class="col" id="header"></th>
+               <th class="col" id="header"></th>
+               <th class="col" id="header">{{ "Drug"|xlt }}</th>
+               <th class="col" id="header">{{ "RxNorm"|xlt }}</th>
+               <th class="col" id="header">{{ "Created Changed"|xlt }}</th>
+               <th class="col" id="header">{{ "Dosage"|xlt }}</th>
+               <th class="col" id="header">{{ "Qty"|xlt }}</th>
+               <th class="col" id="header">{{ "Unit"|xlt }}</th>
+               <th class="col" id="header">{{ "Refills"|xlt }}</th>
+               <th class="col" id="header">{{ "Provider"|xlt }}</th>
+               <th class="col" id="header"></th>
+           </tr>
+
+        </table>
```

Comment:
Hi @juggernautsei 

I suggest the table can remove `width=100`, also if possible, please use `<div class='table-responsive'></div>` to wrap the table, it will let the table have RWD.

Reply:
@stu01509 After talking with gutiersa. I am going to move this to a full tab and yes, I will replace the width.